src/featureFlag/script.js

```diff
@@ -0,0 +1,28 @@
+let applicationState = 'ADD_FLAG'; // ADD_FLAG UPDATE_FLAG
+
+// initialize
+
+async function initialize() {
+  const flags = await fetchFlags();
+  configureListeners();
+  renderFlags(flags);
+  handleOnSubmit();
+}
+
+initialize();
+
+// On selecting flag
+
+// On selecting Add button
```

Comment:
Are these functionalities not defined yet ??

Reply:
Yes. Planning on doing these on incremental basis